### DISCUSSION TITLE
NVIDIA CUDA Virtual Connect With Experts

### DISCUSSION THREAD

**Discussion by jrhemstad** [NVIDIA/cccl#2454] [Announcements]
# NVIDIA CUDA Virtual Connect With Experts Event Information
<div align="center"><img src="https://github.com/NVIDIA/accelerated-computing-hub/assets/172857182/0f315e16-cacd-44f5-9a77-fc62367b05cf" width="600"/></div>

We are starting a monthly virtual event where CUDA enthusiasts can connect directly with a panel of real-life CUDA developers at NVIDIA. This is a chance to meet some of the people designing and implementing your favorite CUDA drivers, kernels, libraries and tools; and ask questions related to your project or work.

# Event Information:

## When? 

**Friday, September 27, 2024 at 10-11:30am PT**

## What?

This month, we will be discussing the **CUDA Python ecosystem**. You will have the opportunity to ask questions via chat about topics like CuPy, Numba, and other Python libraries to our **LIVE** panel of CUDA Python experts.

## Who can Attend? 

CUDA and Python developers of all levels and backgrounds.

## Where? 

Join us on NVIDIA’s Microsoft Teams Instance [here](https://teams.microsoft.com/l/meetup-join/19%3ameeting_YzgwY2EzMWMtYTA0Zi00NDhjLTk0MmMtN2Y4MDRlMjQ2MTI2%40thread.v2/0?context=%7b%22Tid%22%3a%2243083d15-7273-40c1-b7db-39efd9ccc17a%22%2c%22Oid%22%3a%22bc6a8639-bf95-4464-af3e-20c110ea129f%22%7d)

**Comment by zobinHuang** [comment]
Hi! Thanks for organizing this event.

Will the topic of every week's meeting be different? Is there an agenda for the following weeks? 

Thanks :)
